draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -194,99 +194,45 @@ Figure: SD-JWT Issuance and Presentation Flow
 
 # Concepts
 
-In the following, the contents of SD-JWTs and Disclosures are described at a
-conceptual level, abstracting from the data formats described afterwards.
+This section describes SD-JWTs and Disclosures at a
+conceptual level, abstracting from the data formats described in (#data_formats).
 
-## Creating an SD-JWT
+## SD-JWT and Disclosures
 
-An SD-JWT, at its core, is a digitally signed document containing digests over the claims (per claim: a random salt, the claim name and the claim value).
-It MAY further contain clear-text claims that are always disclosed to the Verifier.
-It MUST be digitally signed using the Issuer's private key.
+An SD-JWT, at its core, is a digitally signed document containing digests of the selectively discloseable claims (per claim: a random salt, the claim name and the claim value) with the clear-text claims as Disclosures outside the document.
```

Comment:
```suggestion
An SD-JWT, at its core, is a digitally signed JSON document containing digests over the selectively disclosable claims with the Disclosures outside the document.
```

Reply:
- I don't understand what `the clear-text claims as Disclosures` means
- `(per claim: a random salt, the claim name and the claim value)` was also confusing to me.